Formula/cadence-workflow.rb

```diff
@@ -0,0 +1,33 @@
+class CadenceWorkflow < Formula
+  desc "Cadence Workflow is a distributed, scalable, durable, and highly available orchestration engine to execute asynchronous long-running business logic in a scalable and resilient way"
+  homepage "https://cadenceworkflow.io/"
+  url "https://github.com/uber/cadence.git",
+      tag:      "v0.18.1",
+      revision: "b6dfe37ebfec7742b1470be7129f07518e8bec98"
+  license "MIT"
+
+  bottle do
+  end
+
+  depends_on "go" => :build
+
+  conflicts_with "cadence", because: "both install an `cadence` executable"
+
+  def install
+    ENV["PATH"] = "#{ENV["PATH"]}:#{ENV["GOPATH"]}/bin"
+    system "make", "cadence", "cadence-server", "cadence-canary", "cadence-sql-tool", "cadence-cassandra-tool"
+    bin.install "cadence"
+    bin.install "cadence-server"
+    bin.install "cadence-canary"
+    bin.install "cadence-sql-tool"
+    bin.install "cadence-cassandra-tool"
```

Comment:
Is there no `make install`? We should use that.

Reply:
No, but there is not such target: see https://github.com/uber/cadence/blob/master/Makefile